sql/updates/world/4.3.4/2022_04_10_01_world.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `conditions` WHERE `ConditionTypeOrReference`=1 AND `ConditionValue1`=58984;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES (17, 0, 6785, 0, 0, 1, 0, 58984, 0, 0, 1, 0, 0, '', 'Druid - Ravage cannot be used while Shadowmeld is active');
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES (17, 0, 9005, 0, 0, 1, 0, 58984, 0, 0, 1, 0, 0, '', 'Druid - Pounce cannot be used while Shadowmeld is active');
```

Comment:
Get rid of these and move them to SpellScript, using the CheckCast handling.  Also don't use  single inserts for multiple entries, merge them all into one insert query. Like here:  https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/blob/master/sql/updates/world/4.3.4/2022_03_10_01_world.sql